test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -599,6 +652,10 @@ class test_uct_cm_sockaddr : public uct_test {
     volatile int           m_server_recv_req_cnt, m_client_connect_cb_cnt,
                            m_server_connect_cb_cnt;
     volatile int           m_server_disconnect_cnt, m_client_disconnect_cnt;
```

Comment:
this class looks overcomplicated, maybe even split it for 3?
```
cm_base -> cm_simple
    \
       -> cm_stress
```

Reply:
This file would have four classes only for sockaddr cm testing..

Reply:
I don't see the problem if they have logical inheritance